packages/neocmakelsp/package.yaml

```diff
@@ -10,7 +10,18 @@ categories:
   - LSP
 
 source:
-  id: pkg:cargo/neocmakelsp@0.7.9
+  id: pkg:github/Decodetalkers/neocmakelsp@v0.7.9
+  asset:
+    - target: darwin_arm64
+      file: neocmakelsp-aarch64-apple-darwin
+    - target: darwin_x64
+      file: neocmakelsp-x86_64-apple-darwin
+    - target: win_x64
+      file: neocmakelsp-x86_64-pc-windows-msvc.exe
+    - target: linux_x64_gnu
+      file: neocmakelsp-x86_64-unknown-linux-gnu
+    - target: linux_x64_musl
+      file: neocmakelsp-x86_64-unknown-linux-musl
 
```

Comment:
It supports aarch64 for linux since [v0.8.12.2](https://github.com/neocmakelsp/neocmakelsp/releases/tag/v0.8.12.2).